code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -954,6 +954,11 @@
 	gib()
 	return
 
+/mob/living/silicon/robot/update_canmove() // to fix lockdown issues w/ chairs
+    if(lockcharge)
+        canmove = 0

```

Comment:
I always use tabs. I searched my copy of the code and there are no leading spaces anywhere. Unless the regex `^    ` wouldn't work, for some reason, but I'm sure it does.